ERCS/erc-7208.md

```diff
@@ -0,0 +1,598 @@
+---
+eip: 7208
+title: On-Chain Data Container
+description: Abstracting logic away from storage for RWA Tokenization
+author: Rachid Ajaja <rachid@allianceblock.io>, Alexandros Athanasopulos (@Xaleee), Pavel Rubin (@pash7ka), Sebastian Galimberti Romano (@galimba)
+discussions-to: https://ethereum-magicians.org/t/erc-7208-on-chain-data-container/14778
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-06-09
+requires: 721
+---
+
+## Abstract
+
+- This ERC defines several interface for ODCs (On-Chain Data Containers) that together allow for the storage and retrieval of additional on-chain data, referred to as "Properties". This provides an interoperable solution for dynamic data management across various token standards, enabling them to hold mutable states and reflect changes over time.
+
+- ODCs aim to address the limitations of both previous and future ERCs by enabling on-chain data aggregation, providing an interface for standardized, interoperable, and composable data management solutions.
+
+- ODCs address some of the existing limitations with previous ERC token contracts by separating the logic from the storage.
+
+- This ERC defines a standard interface for interacting with the concept of "Restrictions" for data stored within an ODC ("Properties"). This enables greater flexibility, interoperability, and utility for multiple ERCs to work together.
+
+[ODC Overview HERE]
```

Comment:
```suggestion
<!-- ODC Overview HERE -->
```

If you use HTML-style comments, the linter will make sure you remove them before moving to review.